README.md

```diff
@@ -55,9 +55,9 @@ record HelloList(List<HelloWorld> hellos) implements Component {
   }
 }
 ```
-
-Here are 3 more demos ([Demo.java](test/main/java/Demo.java), [Demo2.java](test/main/java/Demo2.java) and
-[DemoList.java](test/main/java/DemoList.java)).
+https://github.com/forax/html-component/blob/master/src/test/java/Demo.java
```

Comment:
Hello, I would remove this line.
I don't know if you keep it by mistake but i would prefer inline links like the ones below instead of a full URL.

Reply:
Yes, sort it was a mistake